ospd/vts.py

```diff
@@ -53,6 +53,9 @@ def __iter__(self) -> Iterator[str]:
     def __getitem__(self, key):
         return self.vts[key]
 
+    def items(self):
+        return self.vts.items()
+
```

Comment:
IMHO we can ensure that items already returns an iterator.
```suggestion
    def items(self) -> Iterator[Tuple[str, str]]:
        return iter(self.vts.items())

```